src/console/App.js

```diff
@@ -48,17 +68,22 @@ class App extends Component {
     login();
   };
 
-  onLoggedIn = () => {
-    fetchUserInfo(this.onUserInfo);
+  onLoggedIn = authResult => {
```

Comment:
@rehandalal How do you feel about this notation over using `@autobind` ?

Reply:
ah, yeah, i got used to writing arrow functions everywhere because i like the syntax better and because they do auto bind, but I can rewrite it if needed.

Reply:
I'm indifferent either way, though we should be consistent (currently we pretty much only use `@autobind`)